dev/template-values-development.json

```diff
@@ -5,9 +5,9 @@
     "private": false
   },
   "sanity": {
-    "dataset": "master",
-    "projectId": "rwmuledy",
-    "projectTitle": "Kitchen"
+    "dataset": "kitchensink",
+    "projectId": "jmhf2rn8",
+    "projectTitle": "Kitchen Sink Studio"
```

Comment:
Was this included by mistake? It would mess up my demo setup if we include this. This whole file should probably be in gitignore

Reply:
[I pinged you about this in the slack]

Reply:
Solved with gitignore and a .example file